torch_xla/core/xla_model.py

```diff
@@ -491,8 +491,10 @@ def all_reduce(reduce_type, inputs, scale=1.0, groups=None, pin_layout=True):
     if scale == 1.0 and groups == [] and pin_layout:
       # TODO(alanwaketan): Support groups.
       # Only c10d_functional version cc ops are traceable by Dynamo.
-      result = torch.ops.c10d_functional.all_reduce(inputs, reduce_type, "", [],
-                                                    0)
+      pg = torch.distributed.GroupMember.WORLD
+      assert pg is not None, "The default process group hasn't been initialized"
```

Comment:
We don't always use pg. So no need to have an assertion here.

Reply:
Hmm, this can be called without `torch.distributed.init_process_group`?